schematic/store/synapse.py

```diff
@@ -1653,7 +1692,41 @@ def _create_entity_id(self, idx, row, manifest, datasetId):
         manifest.loc[idx, "entityId"] = entityId
         return manifest, entityId
 
-    def add_annotations_to_entities_files(
+    async def _store_annos(self, requests):
```

Comment:
nit: Typing for arguments and return.

I think it's the `pylance` extension that I have installed that will help point these out automatically for me. This is an example of a random file I opened:

![image](https://github.com/Sage-Bionetworks/schematic/assets/17128019/0cb5f32f-45cd-416b-b45b-04c0ac69b81e)